src/idlcxx/CMakeLists.txt

```
@@ -21,7 +21,8 @@ add_library(
 set_target_properties(idlcxx PROPERTIES
    OUTPUT_NAME "cycloneddsidlcxx"
    VERSION ${PROJECT_VERSION}
-   SOVERSION ${PROJECT_VERSION_MAJOR})
+   SOVERSION ${PROJECT_VERSION_MAJOR}
+   C_STANDARD 11)
```

`C_STANDARD 11` is going to be a problem, there are too many platforms that don't do C11 yet (it is only a decade old ...), but the code is C99 so this
```suggestion
   C_STANDARD 99)
```
instead would do the trick. Even Microsoft has caught up with the new millennium by now.